### PR TITLE
Enhancement/section styles

### PR DESCRIPTION
**Pull request by suejinkim20**

### CODE REVIEW

**Comment by suejinkim20** [comment]
Issues that will be addressed in future developments:
- remove the top margin of the right text body section (this will be addressed when the custom Markdown component is created)
- incorporate the imageText section on the /pages/people/[slug].js page at the top where the staff photos are displayed
- for developer experience, rework how PeopleGrid and PersonCard are structured (this component is now being used on the staff page as well as wherever staff photos are being displayed, such as individual operational team pages, research group pages, etc.

File: components/layout/section/section.js [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
 
 export const Section = ({ title, children }) => {
   return (
-    <Grid
-      container
-      columns={ 8 }
-      columnSpacing={{ xs: 0, md: 2 }}
-      rowSpacing={{ xs: 2, md: 10 }}
+    <Stack
+      direction="row"
```
Comment (mbwatson) [inline components/layout/section/section.js]: let's build responsiveness into this component, and stack the title and body on smaller displays

File: components/layout/section/section.module.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
   display: flex;
```
Comment (mbwatson) [inline components/layout/section/section.module.css:2]: consider moving styles from here to the sx prop. then, i think, we won't have to worry about overrides (!important)

Comment (suejinkim20) [reply components/layout/section/section.module.css:2]: `section.js` isn't actually calling any styles from this file so i think we could delete it altogether?

**Comment by mbwatson** [comment]
can we prevent this from happening, or is that going to be fixed by the Markdown component?
[Screencast from 01-12-2023 04:50:56 PM.webm](https://user-images.githubusercontent.com/6019870/212188847-ef8c9dff-9a4a-4f60-8ba1-518442eea847.webm)

**Comment by suejinkim20** [comment]
> can we prevent this from happening, or is that going to be fixed by the Markdown component? [Screencast from 01-12-2023 04:50:56 PM.webm](https://user-images.githubusercontent.com/6019870/212188847-ef8c9dff-9a4a-4f60-8ba1-518442eea847.webm)

it is not fixed by the Markdown component but having these urls translated to actual links will keep these long lines from being a part of the body of the text. hence, there won't be anything to cause the text box to stretch. however, we should put something in place that limits the size of the textbox for the future. im not sure what we decided before, but would putting a min- or max-width on the text box help this situation, or would a flex-basis be applicable? there is a flex-basis of 200px on the "title" part of the section.

**Comment by mbwatson** [comment]
ok, yeah, that's what i was thinking--that the long strings wouldn't cause such a stretch. 

i think we want a flex-grow of zero on the left-hand side, on large displays. i'll leave a note on the line i'm talking about.

File: components/layout/section/section.js [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+    sx={{
+        margin: '3rem 6rem',
         '& .title': {
           xs: { textAlign: 'left' },
           md: { textAlign: 'right' },
         },
       }}
     >
-      <Grid item xs={ 8 } md={ 3 }>
+      <Box sx={{
```
Comment (mbwatson) [inline components/layout/section/section.js:17]: with a `flex-grow: 0` (or `flex: 0 ...`) the left-hand side won't grow to fill the available space, so try something like this (unsure about MUI/sx syntax :smile:  )

```
      <Box sx={{
        flex: {
          xs: '0 0',
          md: '0 1 200px',
        }
      }}>
```

Comment (mbwatson) [reply components/layout/section/section.js:17]: maybe '0 0 200px' ?

**Review by mbwatson — APPROVED** [review]
:100: nice!

**Pull request merged**